R/make-test.R

```diff
@@ -27,7 +27,26 @@ make_test <- function(elts, opt = demo_options(),
   shiny::shinyApp(
     ui = ui(opt = opt),
     server = server(elts = elts, opt = opt,
-                    custom_admin_panel = custom_admin_panel))
+                    custom_admin_panel = custom_admin_panel),
+    onStart = setup_on_start(opt)
+    )
+}
+
+setup_on_start <- function(opt) {
+
+  function() {
+    # On start:
+    cat("Setting up application \n")
```

Comment:
Is there a particular reason you used `cat` here? For consistency with the rest of psychTestR, can we use `message`, and remove the `\n`?

Reply:
That was just part of the example I adapted from. Updating to `message()`.

Reply:
cool thanks!